docs/install/external-disk-support.md

```diff
@@ -0,0 +1,90 @@
+
+If in doubt, consult the system provider's documentation. An example document link is provided in the References section. Other vendors should provide similar documentation.
+
+The typical things you need to configure are:
+
+* Enable UEFI boot
+* Configure iSCSI initiator and target parameters
+* Enable the iSCSI device in the boot menu
+* Set the boot order so that your system will boot from the iSCSI device
+
+![target-details.png](/img/v1.4/external-disk/target-details.png)
+
+#### install Harvester
+This can be done by whatever means you would normally use to load the Harvester install image.
+
+The Harvester installer should automatically "see" the iSCSI device in the dialog where you chose the installation destination. Choose this device to install.
+
+The following changes have been made to the installer when an iscsi target is used:
+* network interfaces being used for mounting the iscsi volumes are skipped from the installer network configuration page
+* the disks page will show the first path to a multipath'd remote disk, however post install assuming `os.externalStorageConfig` is provided, the OS will boot off the multipath device
+
+During the install phase, users need to provide a config yaml to configure multipath and additional kernel arguments. This information is added to the installed OS to allow subsequent boots from an iscsi target
+
+For example in our environment we have provided the following config.yaml
+
+```
+os:
+  write_files:
+  - content: |
+      name: "fix default gateway"
+      stages:
+        network:
+          - commands:
+            - ip route delete default dev enp4s0f0.2017
+            - ip route add default via 10.115.7.254
+    path: /oem/99_fix_gateway.yaml
+  externalStorageConfig:
+    enabled: true
+    multiPathConfig:
+    - vendor: "IET"
+      product: "MediaFiles"
+  additionalKernelArguments: "rd.iscsi.firmware vlan=enp4s0f0.2017:enp4s0f0 ip=10.115.48.10::10.115.55.254:255.255.248.0::enp4s0f0.2017:none"
+``` 
+
+In our test setup, we have multiple tagged VLANs. With VLAN 2017 used for connecting with the iSCSI volume and VLAN2011 being used for Harvester management interface.
+
+`vlan=enp4s0f0.2017:enp4s0f0 ip=10.115.48.10::10.115.55.254:255.255.248.0::enp4s0f0.2017:none` kernel argument is only needed if the iSCSI volume is accessible via an interface on a tagged VLAN. The arguments ensure that additional tagged interface is created during boot and a static address is allocated to the same.
+
+Please refer to the [dracut cmdline arguments](https://manpages.opensuse.org/Tumbleweed/dracut/dracut.cmdline.7.en.html) to configure the kernel arguments to suit your use case.
+
+The `write_files` directive is needed to ensure that the management interface is used as the default gateway. This is essential as RKE2 uses the interface with the default gateway as the node address. 
+
+Post install Harvester should boot up and work as it would normally.
```

Comment:
```suggestion
The test setup uses multiple tagged VLANs, such as VLAN 2017 (used for connecting with the iSCSI volume) and VLAN 2011 (used for the Harvester management interface).

The kernel argument `vlan=enp4s0f0.2017:enp4s0f0 ip=10.115.48.10::10.115.55.254:255.255.248.0::enp4s0f0.2017:none` is necessary only if the iSCSI volume is accessible via an interface on a tagged VLAN. The arguments ensure that an additional tagged interface is created during boot and that a static address is allocated to the interface. See [dracut.cmdline](https://manpages.opensuse.org/Tumbleweed/dracut/dracut.cmdline.7.en.html) for more information about configuring the kernel arguments to match your use case.

The `write_files` directive is needed to ensure that the management interface is used as the default gateway. This is essential because RKE2 uses the interface with the default gateway as the node address.
```